Gemfile

```diff
@@ -59,7 +59,7 @@ group :development, :test do
   gem 'rubocop', '~> 0.42.0', require: false
 
   # Security scanners
-  gem 'brakeman'
+  gem 'brakeman', '3.4.1'
```

Comment:
Honestly, just habit? For things like Brakeman I, personally, only like to have them bump when I do it explicitly. Happy to change it to `~> 3.4` so that we only have to make a choice when they jump to 4.0